cumulus/parachains/runtimes/assets/asset-hub-rococo/src/lib.rs

```diff
@@ -893,6 +893,37 @@ impl pallet_xcm_bridge_hub_router::Config<ToRococoXcmRouterInstance> for Runtime
 	type FeeAsset = xcm_config::bridging::XcmBridgeHubRouterFeeAssetId;
 }
 
+/// XCM router instance to BridgeHub with bridging capabilities for `Westend` global
+/// consensus with dynamic fees and back-pressure.
+pub type ToWestendXcmRouterInstance = pallet_assets::Instance3;
```

Comment:
I never did like the fact that we're reusing `pallet_assets::Instance(s)` to specify `pallet_xcm_bridge_hub_router<I>` instances - I say fix it here, define some `struct ToRococoRouterInstance` and `struct ToWestendRouterInstance` and use those instead.

Later, Snowfork will also add yet another instance for their bridge and having explicitly named structs is better.

Reply:
wow, totally typo, I've never realized that we use `pallet_assets` here, omg, I am going to fix it,